client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -72,24 +74,26 @@ export default function ItemPreviewPaneHeader( {
 							</Button>
 						</div>
 					</div>
-					<div className="item-preview__header-actions">
-						{ extraProps?.headerButtons ? (
-							<extraProps.headerButtons
-								focusRef={ focusRef }
-								itemData={ itemData }
-								closeSitePreviewPane={ closeItemPreviewPane || ( () => {} ) }
-							/>
-						) : (
-							<Button
-								onClick={ closeItemPreviewPane }
-								className="item-preview__close-preview"
-								aria-label={ translate( 'Close Preview' ) }
-								ref={ focusRef }
-							>
-								<Gridicon icon="cross" size={ ICON_SIZE_REGULAR } />
-							</Button>
-						) }
-					</div>
+					{ isPreviewLoaded && (
```

Comment:
What is the issue we are fixing here? It's to understand well this conditional

Reply:
It's for https://github.com/Automattic/wp-calypso/pull/91100#pullrequestreview-2080035487

Reply:
👍  Thank you for the explanation. I didn't see that comment.